cedar-policy-core/src/parser/text_to_cst.rs

```diff
@@ -335,8 +335,7 @@ mod tests {
         assert!(policy.is_ok());
     }
 
-    #[test]
-    #[should_panic] // we no longer support structs
+    #[test] // we no longer support structs
```

Comment:
what does this comment mean

Reply:
I assume it means at some point we supported named structs like `one{num:true,trivia:"first!"}` but we don't anymore.